test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -570,7 +570,9 @@ TEST(LiteInterpreterTest, HierarchyModuleInfo) {
   // "top(C).B0(B).forward": for the add operator in B0.
   // "top(C).B0(B).A0(A).forward": for the add operator in A0.
```

Comment:
These comments probably need to be updated or make comments inline (within the set maybe?) to be consistent with the changes below.